common/recommender/Baseline.js

```diff
@@ -35,11 +35,13 @@ class Baseline {
   extractFeatures(entry) {
     const urlObj = URL(entry.url);
     const host = urlObj.host;
+    // For empty profiles.
     const occurrences = this.domainCounts.get(host) || 1;
+    const domainCountsSize = this.domainCounts.size || 1;
+    const tf = entry.visitCount || 1;
+    const idf = Math.log(domainCountsSize / occurrences) || 1;
```

Comment:
I'm not sure whether or not 1 is a good default idf here. Consider using the "idf smooth" instead?

[Wikipedia TF-IDF](https://en.wikipedia.org/wiki/Tf%E2%80%93idf)